tock/tock/templates/uaa_client/login_error.html

```diff
@@ -1,22 +1,20 @@
 {% extends "base.html" %}
-{% block content %}
 
-<h2>cloud.gov Login Error</h2>
+{% block content %}
 
-{% if error_code == "missing_state" or
-      error_code == "missing_session_state" or
-      error_code == "invalid_state" %}
+  <h2>cloud.gov Login Error</h2>
 
-<p>Bad session state</p>
+  {% if "_state" in error_code %}
```

Comment:
Huh, where does `_state` come from?

Reply:
We're substring matching. It's in all the of the state error_codes, so it's a fast way to check, without having to explicitly write them all out like we did before.

Reply:
Oh LOL, my bad, my brain was in JavaScript-land.